Audio Player crashes on rotate

New audio player for next version is crashing in landscape mode

App version: 1.5.0.x

Stacktrace: 
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{de.danoeh.antennapod/de.danoeh.antennapod.activity.AudioplayerActivity}: java.lang.IllegalArgumentException: No view found for id 0x7f0e00b1 (de.danoeh.antennapod:id/pager) for fragment CoverFragment{3642ae4 #0 id=0x7f0e00b1}
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2432)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2492)
	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4093)
	at android.app.ActivityThread.-wrap15(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1366)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:5433)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: java.lang.IllegalArgumentException: No view found for id 0x7f0e00b1 (de.danoeh.antennapod:id/pager) for fragment CoverFragment{3642ae4 #0 id=0x7f0e00b1}
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1059)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1248)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1230)
	at android.support.v4.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:2042)
	at android.support.v4.app.FragmentController.dispatchActivityCreated(FragmentController.java:165)
	at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:543)
	at de.danoeh.antennapod.activity.MediaplayerActivity.onStart(MediaplayerActivity.java:244)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1237)
	at android.app.Activity.performStart(Activity.java:6268)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2395)
	... 10 more
```

--- a/app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java
+++ b/app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java
@@ -257,12 +257,6 @@ public abstract class MediaplayerActivity extends AppCompatActivity implements O
         }
     }
 
-    @Override
-    protected void onDestroy() {
-        super.onDestroy();
-        Log.d(TAG, "onDestroy()");
-    }
-
     @TargetApi(Build.VERSION_CODES.ICE_CREAM_SANDWICH)
     @Override
     public void onTrimMemory(int level) {

--- a/app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java
+++ b/app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java
@@ -48,7 +48,6 @@ public class CoverFragment extends Fragment implements
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setRetainInstance(true);
         Bundle args = getArguments();
         if (args != null) {
             media = args.getParcelable(ARG_PLAYABLE);

